web/packages/core/src/load-ruffle.ts

```diff
@@ -55,8 +61,40 @@ async function fetchRuffle(config: Config): Promise<typeof Ruffle> {
     const { default: init, Ruffle } = await (extensionsSupported
         ? import("../pkg/ruffle_web-wasm_extensions")
         : import("../pkg/ruffle_web"));
+    let response;
+    const wasmUrl = extensionsSupported
+        ? new URL("../pkg/ruffle_web-wasm_extensions_bg.wasm", import.meta.url)
+        : new URL("../pkg/ruffle_web_bg.wasm", import.meta.url);
+    const wasmResponse = await fetch(wasmUrl.href);
+    if (progressCallback) {
+        const contentLength = wasmResponse.headers.get("content-length")!;
+        let bytesLoaded = 0;
+        const bytesTotal = parseInt(contentLength, 10);
```

Comment:
nit: It's a common good practice to use `Number(...)` over `parseInt()`, since the latter is more lenient. For example `parseInt("123abc")` will parse successfully as `123`, whereas `Number("123abc")` will fail by returning `NaN`.